lib/Dialect/TritonGPU/IR/LinearLayoutConversions.cpp

```diff
@@ -71,14 +71,18 @@ void assertIsRegisterLayout(const LinearLayout &layout) {
                                          expectedOuts.end()));
 }
 
-// Returns a 1D -> ND layout that's equivalent to creating a 1D -> 1D mapping of
-// size product(shape) and then reshaping to permute(shape, order).
+// Returns a 1D -> ND layout into [dim0, dim1, ...] that's equivalent to
+// creating a 1D -> 1D mapping of size product(shape) and then reshaping to
+// permute(shape, order).
 LinearLayout identityND(StringAttr inDimName, ArrayRef<unsigned> shape,
```

Comment:
I think this is more like `identityStandardND`

Reply:
There's also a comment you may want to fix

```
// Returns ["out0", "out1", ..., "out<rank-1>"].
SmallVector<StringAttr> standardOutDimNames(MLIRContext *ctx, int rank) {
```

> // Returns ["dim0", "dim1", ..., "dim<rank-1>"].

Reply:
Fair, will change the name (and fix the comment)